core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerContainerFactory.scala

```diff
@@ -67,13 +70,13 @@ class DockerContainerFactory(config: WhiskConfig,
       image = image,
       userProvidedImage = userProvidedImage,
       memory = memory,
-      cpuShares = config.invokerCoreShare.toInt,
+      cpuShares = poolConfig.cpuShare,
```

Comment:
Philosophically I'm ok with this, but I'm not sure about how to update KubernetesContainerFactory to respect this change; @dgrove-oss WDYT?

For the time being, it may be better to leave this to a separate PR, since it is changing the ContainerFactory SPI interface.

Reply:
on second thought - I'll go ahead and put this in, and if there is a simple way to update kube to respect the cpu limit, that can be added later; for now, kube will do nothing with that parameter.

Reply:
go ahead and put it in;  I will update KubernetesContainerFactory later.  You are right, currently we are only passing in a memory limit in KubernetesContainerFactory.  However, passing in a CPU limit will be small to the change in container creation in #3338